### ISSUE TITLE
methods have gone missing on Joi.any() with new update

### ISSUE THREAD

**Issue by lonewarrior556** [TCMiranda/joi-extract-type#36]
```
    "@hapi/joi": "15.1.1",
    "joi-extract-type": "15.0.2",
    "@types/hapi__joi": "15.0.4",
    "typescript": "3.8.3"
```
![image](https://user-images.githubusercontent.com/4774405/87340264-d498f300-c515-11ea-8afa-9c8ffca01f39.png)

VS

```
    "@hapi/joi": "15.1.1",
    "joi-extract-type": "15.0.7",
    "@types/hapi__joi": "15.0.4",
    "typescript": "3.8.3"
```
![image](https://user-images.githubusercontent.com/4774405/87339903-51779d00-c515-11ea-8b5c-1131873c9c4a.png)

**Comment by lonewarrior556** [comment]
I just noticed that it's strictly on any,. probly something was missed in https://github.com/TCMiranda/joi-extract-type/pull/34
(my bad) will try to see if I can fix

**Comment by TCMiranda** [comment]
Thanks! Published on 15.0.8
